pydantic/_internal/_utils.py

```diff
@@ -387,3 +390,35 @@ def __getitem__(self, key: str, /) -> Any:
 
         def __contains__(self, key: str, /) -> bool:
             return self.wrapped.__contains__(key)
+
+
+_ModelT = TypeVar('_ModelT', bound='BaseModel')
+_R = TypeVar('_R')
```

Comment:
Can we make this typevar more clear, maybe like `PropertyReturnType` or `ReturnType` or something like that?